auto_round/script/mllm.py

```diff
@@ -229,6 +230,16 @@ def setup_lmeval_parser():
     return args
 
 
+def _default_args_check(args):
+    if 'liuhaotian/llava' in args.dataset:
+        args.truncation = False if "--trancation" not in sys.argv else args.truncation
+        args.batch_size = 1 if "--batch_size" not in sys.argv else args.batch_size
+        args.gradient_accumulate_steps = 4 if \
+            "--gradient_accumulate_steps" not in sys.argv else args.gradient_accumulate_steps
+        args.seqlen = 512 if "--seqlen" not in sys.argv else args.seqlen
```

Comment:
why need these two checker, the original code should already handle these issues